industrial_ci/src/builders/catkin_tools_isolated_devel.sh

```diff
@@ -27,5 +27,5 @@ function ici_extend_space {
 function _catkin_config {
     local extend=$1; shift
     local ws=$1; shift
-    ici_exec_in_workspace "$extend" "$ws" catkin config --isolate-devel
+    ici_exec_in_workspace "$extend" "$ws" catkin config --install --link-devel
```

Comment:
Why do you re-introduce `--install` here and remove `--isolate-devel`?
The builder name suggests an _isolated_ devel-space _only_.

Reply:
The idea is to test the install tags anyway, even if they will not used in the chained workspaces.
`--isolate-devel` and `--install` together render the `devel` [unusable](https://github.com/catkin/catkin_tools/blob/f5a4a0ade82eb02891727b390c59cd1b6eef6f85/catkin_tools/verbs/catkin_build/build.py#L585).

Reply:
I don't agree here. I think we definitely want `--isolate-devel`. If that's incompatible to `--install` (I don't yet understand why), we should drop `--install` (It's not used anyway as you claimed).

Reply:
> I don't yet understand why

Looks like this is an open issue (https://github.com/catkin/catkin_tools/issues/674#issuecomment-879008090)
I removed it for now.

Reply:
This feature (?) was implemented in https://github.com/catkin/catkin_tools/pull/24, so apparently sourcing the isolated devel would source the isolated install spaces anyway (except for the leaf packages, I guess).